### PR TITLE
Refresh: Homepage strings (#15306)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Convert homepage strings to ftl identifiers.

## Issue / Bugzilla link

 #15306

## Testing

_Note: I have not used the gallery strings because I wanted to avoid a conflict with #15481_

### CODE REVIEW

File: bedrock/mozorg/views.py [hunk 1]
```diff
@@ -129,9 +129,9 @@ class HomeView(L10nTemplateView):
     m24_template_name = "mozorg/home/home-m24.html"
     template_name = "mozorg/home/home-new.html"
     old_template_name = "mozorg/home/home-old.html"
-    activation_files = ["mozorg/home-new", "mozorg/home"]
+    activation_files = ["mozorg/home-m24", "mozorg/home-new", "mozorg/home"]
 
-    ftl_files_map = {old_template_name: ["mozorg/home"], template_name: ["mozorg/home-new"]}
+    ftl_files_map = {old_template_name: ["mozorg/home"], template_name: ["mozorg/home-new"], m24_template_name: ["mozorg/home-m24"]}
```
Comment (stephaniehobson) [inline bedrock/mozorg/views.py:134]: A few lines down this is missing the code necessary to active this homepage if it is translated in languages other than English.

File: l10n/en/mozorg/home-m24.ftl [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+m24-home-type = Type
+m24-home-author = Author
+m24-home-topic = Topic
+m24-home-intro = Intro
+m24-home-tag-article = Article
+m24-home-tag-podcast = Podcast
+m24-home-tag-video = Video
+m24-home-topic-news = News
+m24-home-topic-ai = Artificial Intelligence
+m24-home-topic-ps = Privacy &amp; Security
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:72]:
```suggestion
m24-home-topic-ps = Privacy & Security
```

File: l10n/en/mozorg/home-m24.ftl [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+m24-home-keeping-genai-technologies = Keeping GenAI technologies secure is a shared responsibility.
+m24-home-from-hollywood-to = From Hollywood to hip hop, artists are negotiating AI consent.
+m24-home-mozilla-celebrates-groundbreaking = { -brand-name-mozilla } celebrates groundbreaking creators in new docuseries “{ -brand-name-firefox } Presents.”
+m24-home-big-techs-power = Big tech’s power over language means power over people. Bridget Todd talks to language community leaders paving the way for voice AI in their own languages and dialects.
+m24-home-why-does-it = Why does it so often feel like we’re part of a mass AI experiment? What is the responsible way to test new technologies? Bridget Todd explores what it means to live with unproven AI systems that impact millions of people as they roll out across public life.
+m24-home-kids-are-growing = Kids are growing up in a very online world. What’s a concerned parent to do?
+
+## Showcase
+
+m24-home-rise25-awards = Rise25 Awards
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:84]: **Question (non-blocking):** Is it worth making `Rise25` a brand name variable at this point?

File: l10n/en/mozorg/home-m24.ftl [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+m24-home-get-to-know = Get to know the leaders championing trustworthy, inclusive and transparent AI development as they set a new standard and inspire the next generation of innovators.
+m24-home-watch-the-award = Watch the award show
+
+## Issues
+
+m24-home-explore-issues-shaping = Explore issues shaping the future of the internet
+m24-home-privacy-not-included = *Privacy Not Included
+m24-home-search-our-reviews = Search our reviews to see which tech gadgets and apps are stealthily sneaking your data.
+m24-home-pni-search-now = Search now
+m24-home-irl-podcast = IRL Podcast
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:94]: **Question (non-blocking):** Should `IRL Podcast` also be a brand name?

**Review by alexgibson — CHANGES_REQUESTED** [review]
A few nits/suggestions and a couple of missing strings, but otherwise everything looks good. 

We should probably hold off merging this until we include the gallery strings, so that we can avoid showing mixed content for pages that get translated quickly.

File: l10n/en/mozorg/home-m24.ftl [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+m24-home-run-large-language = Run large language models (LLMs) easily on your computer with llamafile — no installation needed, and your data stays safe on your device.
+m24-home-get-started = Get started
+m24-home-turn-your-ai = Turn your AI idea into reality with our 12-week accelerator. Get funding, mentorship and a chance to pitch your innovation to investors.
+m24-home-read-more = Read more
+m24-home-lumigator = Lumigator
+m24-home-find-the-just = Find the “just right” LLM for your needs, use case and data — like Goldilocks, but with sharper teeth.
+m24-home-can-ai-be = Can AI be trustworthy?
+m24-home-in-a-world = In a world where AI innovation is driven by a select few, we’re risking monopolization of this technology. Applying open-source concepts to AI could change that.
+m24-home-watch-now = Watch now
+m24-home-mozilla-ventures = { -brand-name-mozilla } Ventures
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:55]: ✅ I also updated the navigation to use the brand term

File: bedrock/mozorg/templates/mozorg/home/includes/m24/media-springboard.html [hunk 6]
```diff
@@ -51,12 +51,12 @@ <h3 class="m24-c-intro-title m24-t-md">You, AI and the internet — what’s rea
...
     ) }}
     {{ springboard_item(
-      link_url="https://irlpodcast.org/season7/episode5/",
-      type='Podcast',
+      link_url="https://irlpodcast.org/season7/episode5/" + utm_params,
+      type=ftl('m24-home-tag-podcast'),
       type_icon='podcast',
-      topic='Artificial Intelligence',
+      topic=ftl('m24-home-topic-ai'),
       author='IRL Podcast',
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/media-springboard.html:58]: ✅ I'll make it a brand term.

File: l10n/en/mozorg/home-m24.ftl [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+m24-home-got-an-earlystage = Got an early-stage startup? Pitch your company to { -brand-name-mozilla } Ventures and secure funding to drive positive change for the future of AI and the internet.
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:56]: ✅

File: l10n/en/mozorg/home-m24.ftl (hunk 2 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:72]: ✅

File: l10n/en/mozorg/home-m24.ftl (hunk 3 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:84]: ✅ I'll make it a brand term.

File: l10n/en/mozorg/home-m24.ftl (hunk 4 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:94]: ✅ I'll make it a brand term.

File: bedrock/mozorg/templates/mozorg/home/home-m24.html [hunk 8]
```diff
@@ -31,7 +31,7 @@
     {% include 'includes/protocol/navigation/navigation.html' %}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-m24.html:31]: ✅

File: l10n/en/mozorg/home-m24.ftl [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+## Products
+
+m24-home-love-the-internet = Love the internet again
+m24-home-break-free-from = Break free from big tech — our products put you in control of a safer, more private internet experience.
+m24-home-explore-our-products = Explore our products
+m24-home-firefox = { -brand-name-firefox }
+m24-home-get-the-gold = Get the gold standard for browsing with speed, privacy and control.
+m24-home-thunderbird = { -brand-name-thunderbird }
+m24-home-go-chaosfree-with = Go chaos-free with one app for all your emails, calendars and contacts.
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:22]: ✅

**Comment by stephaniehobson** [comment]
Rebased and ready for re-review. The gallery strings are now in use.

File: l10n/en/mozorg/home-m24.ftl [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+m24-home-tag-podcast = Podcast
+m24-home-tag-video = Video
+m24-home-topic-news = News
+m24-home-topic-ai = Artificial Intelligence
+m24-home-topic-ps = Privacy & Security
+m24-home-headline-you-ai = Headline: You, AI and the internet — what’s really going on?
+m24-home-introducing-anonym = Introducing { -brand-name-anonym }: Raising the bar for privacy-preserving digital advertising.
+m24-home-keeping-genai-technologies = Keeping GenAI technologies secure is a shared responsibility.
+m24-home-from-hollywood-to = From Hollywood to hip hop, artists are negotiating AI consent.
+m24-home-mozilla-celebrates-groundbreaking = { -brand-name-mozilla } celebrates groundbreaking creators in new docuseries “{ -brand-name-firefox } Presents.”
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:87]: **Suggestion (non-blocking):** The term `docuseries` is common online today, but I'm not sure if it's recognized widely as a real word? (my spell checker doesn't seem to recognize it at least). 

Is it worth add an L10n comment to say it's short for "documentary series", or just using the two words for a simplified English string?

File: l10n/en/mozorg/home-m24.ftl [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+m24-home-love-the-internet = Love the internet again
+m24-home-break-free-from = Break free from big tech — our products put you in control of a safer, more private internet experience.
+m24-home-explore-our-products = Explore our products
+m24-home-firefox = { -brand-name-firefox }
+m24-home-get-the-gold = Get the gold standard for browsing with speed, privacy and control.
+m24-home-thunderbird = { -brand-name-thunderbird }
+m24-home-go-chaosfree-with = Simplify your life with one app for all your emails, calendars and contacts.
+m24-home-fakespot = { -brand-name-fakespot }
+# "fake laughs" is a joke and the ;) is a winking face to hint it's a joke. You can translate this in a away that keeps both the repetition and the joke: fake reviews, fake products, fake ___. Or leave the joke and winking face out
+m24-home-spot-fake-reviews = Spot fake reviews, products and maybe one day, even fake laughs ;)
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:31]: **Suggestion: (non-blocking)** this one could be another good contender for en en-US only string, as I'm not entirely sure other languages will get this joke well enough to translate it?

File: l10n/en/mozorg/home-m24.ftl [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+m24-home-get-started = Get started
+m24-home-join-the-movement = Join the movement:<br> AI for the people
+m24-home-our-mission-is = Our mission is to make it easy for people to build with, and collaborate on, open-source, trustworthy AI.
+m24-home-explore-mozilla-ai = Explore { -brand-name-mozilla-ai }
+m24-home-llamafile = { -brand-name-llamafile }
+m24-home-run-large-language = Run large language models (LLMs) easily on your computer with { -brand-name-llamafile } — no installation needed, and your data stays safe on your device.
+m24-home-mozilla-builders= { -brand-name-mozilla } Builders
+m24-home-turn-your-ai = Turn your AI idea into reality with our 12-week accelerator. Get funding, mentorship and a chance to pitch your innovation to investors.
+m24-home-lumigator = { -brand-name-lumigator}
+m24-home-find-the-just = Find the “just right” LLM for your needs, use case and data — like Goldilocks, but with sharper teeth.
```
Comment (alexgibson) [inline l10n/en/mozorg/home-m24.ftl:61]: **Question:** is Goldilocks well known throughout the world - will everyone get the joke? (I would assume yes, but I feel like I am also probably ignorant here).

**Review by alexgibson — APPROVED** [review]
We need to remember to include a config here for Pontoon (also for the about page): https://github.com/mozilla-l10n/www-l10n/pull/454#issuecomment-2503198462

I spotted a couple more strings where I question if they will translate well enough, and may benefit from a simplified English version. But I will leave this up to you.

r+wc

File: l10n/en/mozorg/home-m24.ftl (hunk 11 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:31]: ✅

File: l10n/en/mozorg/home-m24.ftl (hunk 12 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:61]: ✅ It's hard to fight the urge to add the Oxford comma while I'm in here...

File: l10n/en/mozorg/home-m24.ftl (hunk 10 above)
Comment (stephaniehobson) [reply l10n/en/mozorg/home-m24.ftl:87]: ✅

File: l10n/en/mozorg/home-m24.ftl (hunk 9 above)
[the remark this answers is not in the document]
Comment (janbrasna) [reply l10n/en/mozorg/home-m24.ftl]: So the jokes are effectively kept out of en-CA & en-GB this way, only good enough for en-US, right? ;) 

Nah, sorry, kidding aside — I wanted to ask about the string id rule here — when the `en` vs. `en-US` content differs: should the ftl id be made up of the fancier en-US text, or as the simplified English is used as a base for translations it should match the easy string? In this case:
- `m24-home-go-chaosfree-with` = Simplify your life with one app for all your emails…
or
- `m24-home-simplify-your-life` = Simplify your life with one app for all your emails…

(or it doesn't really matter in what locale the text matches?)

File: l10n/en/mozorg/home-m24.ftl (hunk 5 above)
[the remark this answers is not in the document]
Comment (janbrasna) [reply l10n/en/mozorg/home-m24.ftl]: BTW aren't all these newer brands a good candidate for `fluent_linter` additions?

https://github.com/mozilla/bedrock/blob/5cf0a9091c8d088bfae84f11a05f99be84d2fb90/.github/l10n/linter_config.yml#L22-L27

**Pull request merged**